tests/contrib/aiomysql/test_aiomysql.py

```diff
@@ -0,0 +1,181 @@
+import asyncio
+import time
+
+import aiomysql
+
+from ddtrace import Pin
+from ddtrace.contrib.aiomysql import patch
+from ddtrace.contrib.aiomysql import unpatch
+from tests.contrib.asyncio.utils import AsyncioTestCase
+from tests.contrib.asyncio.utils import mark_asyncio
+from tests.contrib.config import MYSQL_CONFIG
+from tests.opentracer.utils import init_tracer
+from tests.subprocesstest import run_in_subprocess
+from tests.utils import assert_is_measured
+
+
+AIOMYSQL_CONFIG = dict(MYSQL_CONFIG)
+AIOMYSQL_CONFIG["db"] = AIOMYSQL_CONFIG["database"]
+del AIOMYSQL_CONFIG["database"]
+
+
+class AiomysqlTestCase(AsyncioTestCase):
```

Comment:
We'd like to do more pytest-style testing going forward in the library. https://github.com/DataDog/dd-trace-py/blob/1.x/tests/contrib/asyncpg/test_asyncpg.py is a good example of a database library that we test with pytest-style fixtures.

Not a super big deal if you're not familiar with pytest but would be nice to have here 🙂